remotes/docker/config/hosts.go

```diff
@@ -490,7 +489,7 @@ func loadCertFiles(ctx context.Context, certsDir string) ([]hostConfig, error) {
 	}
 	hosts := make([]hostConfig, 1)
 	for _, f := range fs {
-		if !f.IsDir() {
+		if f.IsDir() {
```

Comment:
on the prior continue when not a dir ... `:-O`... thx for the fix.

On the continue when it is a dir .. does docker support the use of subdirectories should we walk here?

Reply:
> does docker support the use of subdirectories should we walk here?

No (or at least if it does its not intentional 😅).

Reply:
Thanks for finding this and adding tests :)